whitelist.h

```diff
@@ -1,4 +1,6 @@
 static const char *whitelist[] = {
+    "platform-lookaside.fbsbx.com", // Facebook profile images 1
```

Comment:
this alone is sufficient to view the profile pictures of Facebook friends

Reply:
Without `scontent*.fbcdn.net` one of my friends image is not loading...
![spot1](https://user-images.githubusercontent.com/20153302/86524293-14f2d400-be79-11ea-80e2-368b3b553359.jpg)

And if I check Facebook friends followings who follow other Facebook accounts this is the result without  `scontent*.fbcdn.net`
![spot2](https://user-images.githubusercontent.com/20153302/86524369-386a4e80-be7a-11ea-9564-d81cf97518c3.jpg)